.gitattributes

```diff
@@ -0,0 +1,8 @@
+#
+# https://help.github.com/articles/dealing-with-line-endings/
+#
+# Linux start script should use lf
+/gradlew        text eol=lf
+
+# These are Windows script files and should use crlf
+*.bat           text eol=crlf
```

Comment:
If we don't support `Windows`, please remove these two lines.

Reply:
updated